src/api.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export const getArticles = async () => {
+    const response = await axios.get(
+        'https://news-app-8vge.onrender.com/api/articles'
+    )
+    return response.data;
```

Comment:
This is really nice, remember when you come to have multiple API requests later down the line it's possible to create an _axios instance_ to shorten down the code a bit